src/components/VideoControls/VideoControls.js

```diff
@@ -33,18 +32,26 @@ const VideoControls = (props) => {
             video.pause();
         }
     };
+    const stopVideo = () => (video.currentTime = video.duration);
 
     useEffect(() => {
         video.addEventListener('loadedmetadata', () => {
+            const timeLeftOff = localStorage.getItem(videoName);
+            if (timeLeftOff !== null) {
+                video.currentTime = Number(timeLeftOff);
+            }
+
             setTotalDuration(secondsToTime(video.duration));
         });
         video.addEventListener('timeupdate', () => {
             setRealTime(secondsToTime(video.currentTime));
             const progressWidth = (video.currentTime / video.duration) * 100;
             progress.current.style.width = `${progressWidth}%`;
+            localStorage.setItem(videoName, JSON.stringify(video.currentTime));
```

Comment:
we don't need to use JSON.stringify here. It may falsely give the impression that JSON data is being saved

```suggestion
            localStorage.setItem(videoName, String(video.currentTime));
```